src/main/java/com/twilio/oai/api/RubyApiResourceBuilder.java

```diff
@@ -102,4 +117,34 @@ private List<Resource> getDependentInfo(List<Resource> dependents) {
     private void updateListPath() {
         if (listPath != null) listPath = listPath.replace("${", "#{@solution[:").replace("}", "]}");
     }
+
+    private void updateRequiredPathParams(ApiResourceBuilder apiResourceBuilder) {
+        if (!apiResourceBuilder.requiredPathParams.isEmpty()) {
+            for (CodegenParameter param : apiResourceBuilder.requiredPathParams) {
+                param.vendorExtensions.put("isInstanceParam", !param.paramName.equals("account_sid"));
```

Comment:
This might not be needed since we are already removing account_sid as required param. See [this](https://github.com/twilio/twilio-oai-generator/blob/main/src/main/java/com/twilio/oai/DirectoryStructureService.java#L136-L146)